vendor/vendor.js

```diff
@@ -9,7 +9,7 @@
 
 var vendor = {
 	"moment.js" : "node_modules/moment/min/moment-with-locales.js",
-	"moment-timezone.js" : "node_modules/moment-timezone/moment-timezone.js",
+	"moment-timezone-with-data.js" : "node_modules/moment-timezone/builds/moment-timezone-with-data.js",
```

Comment:
This should be keep the original name moment-timezone.js We dont if other module if use the vendor called moment-timezone.js